pulse_actions/handlers/route_functions.py

```diff
@@ -0,0 +1,18 @@
+import pulse_actions.handlers.treeherder_buildbot as treeherder_buildbot
+import pulse_actions.handlers.treeherder_resultset as treeherder_resultset
+import logging
+
+logging.basicConfig(format='%(levelname)s:\t %(message)s')
+LOG = logging.getLogger()
+
+def route(data, message, dry_run):
+    if 'action' in data:
+        if data["action"] == "backfill":
+            treeherder_buildbot.on_buildbot_event(data, message, dry_run)
+        elif data["action"] in ["trigger_missing_jobs", "trigger_all_talos_jobs"]:
+            treeherder_resultset.on_resultset_action_event(data, message, dry_run)
+        else:
+            # We don't want to take any action on 'retrigger' or 'cancel' events
+            LOG.debug("We currently only support 'backfill', 'trigger_missing_jobs', 'trigger_all_talos_jobs' for multiple listeners.")
```

Comment:
so currently these are the formats for resultset_action, job_action:
1. https://github.com/mozilla/treeherder/blob/master/schemas/resultset-action-message.json
2. https://github.com/mozilla/treeherder/blob/master/schemas/job-action-message.json

The parameters that are different are ('resultset_id', 'job_id') or ('action')... We can probably divert it to a function depending on ('resultset_id', 'job_id') instead of 'action'. I am not sure which one is better.